.gitignore

```diff
@@ -66,3 +66,6 @@ metastore_db/
 # Spark/metastore files
 spark-warehouse/
 derby.log
+
+# BigQuery/metastore files
+gcp/db_folder/
```

Comment:
What is this? Can it be in a `build` or other temporary folder managed by gradle instead?

Reply:
Yep. Done. It's generated by unit tests similar to `spark-warehouse/` above.